### PR TITLE
Centralize all tests in Makefile for a better DX

### PR DESCRIPTION
**Pull request by marcenacp**
- Previously: to launch all tests, you would have to run them manually by entering the whole command.
- Now: to launch all tests, `make tests`.

### CODE REVIEW

File: python/ml_croissant/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+pylint:
+	pylint **/*.py
+
+pytest:
+	pytest .
+
+pytype:
+	pytype --jobs="$$(grep -c ^processor /proc/cpuinfo)" --config=./pyproject.toml
+
+# Launch all tests as launched on GitHub actions by order of importance.
```
Comment (ccl-core) [inline python/ml_croissant/Makefile:17]: Why do we need to launch them by order of importance?

**Review by ccl-core — APPROVED** [review]
Cool, thanks!

**Comment by marcenacp** [comment]
Thanks, @ccl-core!

File: python/ml_croissant/Makefile (hunk 1 above)
Comment (marcenacp) [reply python/ml_croissant/Makefile:17]: Edit: I messed up and merged before pushing the changes: https://github.com/mlcommons/croissant/pull/172

**Pull request merged**